src/style.css

```diff
@@ -1,3 +1,98 @@
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.MoyoHeader__nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100%;
+  flex-grow: 1;
+  align-items: stretch;
+  justify-content: center;
+}
+
+.MoyoHeader {
+  display: flex;
+  width: 100%;
+  height: 60px;
+
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  height: 100%;
+  cursor: pointer;
+  align-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  text-decoration: none;
+}
+
+.MoyoHeader__logo {
+  margin-left: 50px;
+  border-color: blue;
+  font-size: 0;
+}
+
+.MoyoHeader__nav {
+  height: 100%;
+  margin-right: 50px;
+  justify-content: center;
+}
+
+.MoyoHeader__nav__links {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100%;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: center;
+}
+
+.MoyoHeader__nav__elem {
+  margin-right: 20px;
+  flex-grow: 1;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.MoyoLogo__Image {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
```

Comment:
I don't think this property is necessary

Reply:
unfortunately it is necessary because of some strange img property - it was described in the introductory video